.github/workflows/test-scripts.yml

```diff
@@ -14,6 +14,7 @@ jobs:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         scripts: # Add in alphabetical order
           - how-to-guides/data-versioning/scripts
+          - how-to-guides/ddp-training/scripts
```

Comment:
This is just setup for when GH gets the GPU instances, right?

Reply:
yeah, just a placeholder till then

Reply:
Great!